docs/rules/require-meta-docs-description.md

```diff
@@ -0,0 +1,41 @@
+# require rules to implement a meta.docs.description property (require-meta-docs-description)
+
+Defining a clear and consistent description for each rule helps developers understand what they're used for.
+
+In particular, each rule description should begin with an allowed prefix:
+* `enforce`
+* `require`
+* `disallow`
```

Comment:
I think this one should probably be configurable with something like a regex (maybe with `enforce`/`require`/`disallow` as the default). As far as I know, `enforce`/`require`/`disallow` is a convention used by the ESLint team but it's not universally adopted.

Reply:
Good point, I'll update shortly.

Reply:
I added an `allowedPrefixes` array option to the rule.

Reply:
I'd really rather this be a regex option -- choosing a set of specific prefixes feels like an oddly specific restriction to impose on users of the rule.

Reply:
@not-an-aardvark sounds good, I switched to a `pattern` option.